packaging/Rakefile

```diff
@@ -21,7 +21,7 @@ task :default => :all
 def wget(urls)
   urls.each do |url|
     mkdir_p CACHE_DIR
-    sh "wget -c -N -P #{CACHE_DIR} #{url}"
+    sh "wget -q -c -N -P #{CACHE_DIR} #{url}"
```

Comment:
Without the quiet flag, wget produces to many lines of output which surpass the travis limit.